### PR TITLE
Visualscript: Add types for get and set.

### PR DESCRIPTION
**Pull request by fire**
As mentioned in other issues Visual Script get and sets are not being typed for properties.

Godot Engine 3.1
Screen Shot 2019-05-25 at 3 17 54 PM

Pull Request

Screen Shot 2019-05-25 at 3 16 28 PM

Object types not being typed is being worked on.

Edited: 

Object types are left for another day.

### CODE REVIEW

**Comment by fire** [comment]
@Chaosus Can you review this and tell me if this is good as is?

**Comment by Chaosus** [comment]
Well I tested it and have no problem - looks like a good change ! But I'm not an expert in visual scripts - I prefer the normal scripts over this.

**Comment by swarnimarun** [comment]
Shouldn't we also be able to add type hint for script variables similar to how optional typing in GDScript does it, not a request for this PR but just a question to probably create a feature request for it?

_I should really look into the GDScript code, guess I am just lazy_

**Comment by swarnimarun** [comment]
Als the PR looks perfect to me, for all my VisualScript system's understanding is worth

**Comment by fire** [comment]
@akien-mga Please review.

File: modules/visual_script/visual_script_func_nodes.cpp [hunk 1]
```diff
@@ -1037,6 +1036,17 @@ PropertyInfo VisualScriptPropertySet::get_input_value_port_info(int p_idx) const
 		}
 	}
 
+	Variant v;
```
Comment (akien-mga) [inline modules/visual_script/visual_script_func_nodes.cpp]: Isn't this unused?

Comment (fire) [reply modules/visual_script/visual_script_func_nodes.cpp]: Will look later.

Comment (fire) [reply modules/visual_script/visual_script_func_nodes.cpp]: Removed.

File: modules/visual_script/visual_script_func_nodes.cpp [hunk 2]
```diff
@@ -1796,14 +1813,13 @@ PropertyInfo VisualScriptPropertyGet::get_input_value_port_info(int p_idx) const
...
 PropertyInfo VisualScriptPropertyGet::get_output_value_port_info(int p_idx) const {
-
-	if (index != StringName()) {
-
-		Variant v;
-		Variant::CallError ce;
-		v = Variant::construct(type_cache, NULL, 0, ce);
-		Variant i = v.get(index);
-		return PropertyInfo(i.get_type(), "value." + String(index));
+	Variant v;
```
Comment (fire) [inline modules/visual_script/visual_script_func_nodes.cpp]: Unused?

**Review by akien-mga — APPROVED** [review]

**Comment by akien-mga** [comment]
Thanks!

**Pull request merged**